samples/test/automlLanguageSentimentModel.v1beta1.test.js

```diff
@@ -36,6 +36,9 @@ const undeployModelId = 'TST5013860858145102878';
 
 const exec = async cmd => (await execa.shell(cmd)).stdout;
 
+console.info('PROJECT_ID >>>');
+console.info(process.env.PROJECT_ID);
```

Comment:
This will be removed as soon as I figure out what our PROJECT_ID is; this is related to another issue.